lib/ui/screens/about.dart

```diff
@@ -0,0 +1,271 @@
+      bottom: PreferredSize(
+        preferredSize: const Size.fromHeight(156.0),
+        child: Container(
+            height: 156.0,
+            child: Padding(
+              padding: EdgeInsets.fromLTRB(16.0, 14.0, 16.0, 9.0),
+              child: Column(children: <Widget>[
+                Row(children: <Widget>[
+                  SizedBox(
+                    width: 64.0,
+                    height: 64.0,
+                    child: Image(image: Svg('images/app_icon.svg')),
+                  ),
+                  SizedBox(width: 20.0),
+                  Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          _appName,
+                          style: Theme.of(context).textTheme.headline5,
+                        ),
+                        SizedBox(height: 4.0),
+                        Text(
+                          tr.aboutPage.versionString(
+                              version: _appVersion, platform: os),
+                          style: Theme.of(context).textTheme.bodyText2,
+                        ),
+                      ]),
+                ]),
+                SizedBox(height: 14.0),
+                Row(children: <Widget>[
+                  Expanded(
+                      child: SizedBox(
+                          height: 40.0,
+                          child: RaisedButton(
+                            child: Row(children: <Widget>[
+                              Icon(RebbleIcons.rocket, size: 25),
+                              SizedBox(width: 8),
+                              Text(tr.aboutPage.sourceCode.toUpperCase()),
+                            ]),
+                            onPressed: () => {launchURL(sourceURL)},
+                          ))),
+                  SizedBox(width: 16),
+                  Expanded(
+                      child: SizedBox(
+                          height: 40.0,
+                          child: CobbleButton(
+                            outlined: false,
+                            label: tr.aboutPage.licenses,
+                            onPressed: () => {showLicensePage(context: context)},
+                          ))),
+                ])
+              ]),
+            )),
+      ),
+      child: SingleChildScrollView(
+        child: Column(
+          children: <Widget>[
+            Card(
+              margin: EdgeInsets.all(16.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Padding(
+                    padding: EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 8.0),
+                    child: Text(
+                      tr.aboutPage.support,
+                      style: Theme.of(context)
+                          .textTheme
+                          .headline6
+                          ?.copyWith(fontSize: 18),
```

Comment:
`headline6` shouldn't be null, I suggest non-null assertion instead. Current code will silently fallback to default style, I prefer if it failed with error as soon as someone mistakenly sets global `headline6` to null.